database.hh

```diff
@@ -588,6 +588,15 @@ private:
     // Caller needs to ensure that column_family remains live (FIXME: relax this).
     // The 'range' parameter must be live as long as the reader is used.
     // Mutations returned by the reader will all have given schema.
+    //
+    // Precondition: if the slice is reversed, the schema must be reversed as well
+    // and `max_memory_for_unlimited_query` must be engaged.
+    // Reversed slices must be provided in the 'half-reversed' format (the order of ranges
+    // being reversed, but the ranges themselves are not).
+    //
```

Comment:
Note: reversed multi-partition reads can be useful, consider the query `SELECT game, user, score FROM tab ORDER BY score ASC PER PARTITION LIMIT 10` to select the 10 lowest scoring users for each game and prepare an advertisement for a coach for them (the table is naturally ordered by descending score since we usually want the top scores). So we should enable those queries too later.

Reply:
They are enabled, just inefficient (they use `make_reversing_reader`)

Reply:
Ok, fine for now.

Reply:
Correction: they are not enabled, they never were.
```
ORDER BY is only supported when the partition key is restricted by an EQ or an IN.
```

Not saying we shouldn't perhaps enable them some time in the future, but at least we don't need to think about breaking potential current users which depend on such queries running today.